bigtable/src/filter_limit_col_family_regex.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Google\Cloud\Samples\Bigtable;
+
+/**
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/bigtable/README.md
+ */
+
+// [START bigtable_filters_limit_col_family_regex]
+use Google\Cloud\Bigtable\BigtableClient;
+use Google\Cloud\Bigtable\Filter;
+
+/**
+ * Create a limiting filter on a column family with a regex
```

Comment:
nit: lets put a newline between the description and `@param` tags
```suggestion
 * Create a limiting filter on a column family with a regex
 *
```